packages/manager/apps/pci-quota/public/translations/quotas/increase/Messages_fr_FR.json

```diff
@@ -8,13 +8,15 @@
   "pci_projects_project_quota_increase_content": "Êtes-vous sûr de vouloir augmenter les quotas de votre projet {{ projectInfo }} ?",
   "pci_projects_project_quota_increase_buy_success_message": "Le bon de commande pour l'ajout de crédits à votre projet est disponible <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ billingUrl }}\">ici<span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a>",
   "pci_projects_project_quota_increase_back_label": "Retour a la page précedente",
-  "pci_projects_project_quota_increase_success_message": "Votre demande d'augmentation de quota a bien été prise en compte. <a class=\"oui-link_icon\" target=\"_blank\" href=\"{{ ticketUrl }}\">Cliquez ici pour suivre son évolution. <span class=\"oui-icon oui-icon-external-link\" aria-hidden=\"true\"></span></a>",
+  "pci_projects_project_quota_increase_success_message": "Votre demande d'augmentation de quota a bien été prise en compte. <link></link>",
+  "pci_projects_quota_link_label": "Cliquez ici pour suivre son évolution",
```

Comment:
It's better to have the link label inside the component

You just need to create a wrapper around OdsLink